src/envoy/http/backend_routing/README.md

```diff
@@ -20,4 +20,18 @@ This filter is designed to strongly integrate with the following filters:
 ## Configuration
 
 View the [backend routing configuration proto](../../../../api/envoy/http/backend_routing/config.proto)
-for inline documentation.
\ No newline at end of file
+for inline documentation.
+
+## Statistics
+
+This filter records statistics.
+
+### Counters
+
+- `denied`: Number of API Consumer requests that are denied due to invalid headers, etc.
```

Comment:
I prefer to have one stat for each error,  can we have
denied_by_no_path
denied_by_no_operation
allowed_by_no_configured_rules

Reply:
Done